server/task/task-memory/src/test/java/org/apache/james/task/eventsourcing/TaskAggregateTest.java

```diff
@@ -47,12 +47,12 @@
     static final Instant timestamp = Instant.parse("2018-11-13T12:00:55Z");
 
     History buildHistory(Function<EventId, Event>... events) {
-        return History.of(
+        return History.of(CollectionConverters.asScala(
```

Comment:
why don't you keep the java-friendly method?

Reply:
Only used in scala except for this test.